### PR TITLE
Add provider equinix and module_name in Equinix Metal TF configs

### PR DESCRIPTION
**Pull request by codinja1188**

### CODE REVIEW

File: examples/fabric-port-connection/versions.tf [hunk 1]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     oci = {
       source  = "oracle/oci"
       version = ">= 4.66.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-oci/fabric-port-connection"
+  }
```
Comment (ocobles) [inline examples/fabric-port-connection/versions.tf]: Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

File: examples/network-edge-device-connection/versions.tf [hunk 2]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     oci = {
       source  = "oracle/oci"
       version = ">= 4.66.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-oci/network-edge-device-connection"
+  }
```
Comment (ocobles) [inline examples/network-edge-device-connection/versions.tf]:
```suggestion
```

File: examples/service-token-metal-to-oci-connection/versions.tf [hunk 3]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     oci = {
       source  = "oracle/oci"
       version = ">= 4.66.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-oci/service-token-metal-to-oci-connection"
+  }
```
Comment (ocobles) [inline examples/service-token-metal-to-oci-connection/versions.tf]:
```suggestion
```

**Review by ocobles — CHANGES_REQUESTED** [review]
Please, review requested changes

**Comment by codinja1188** [comment]
> Please, review requested changes

updated

**Review by ocobles — APPROVED** [review]

**Pull request merged**